core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -102,6 +102,7 @@ object Defaults {
   val LogDeleteDelayMs = 60000
   val LogFlushSchedulerIntervalMs = Long.MaxValue
   val LogFlushOffsetCheckpointIntervalMs = 60000
+  val LogFlushStartOffsetCheckpointIntervalMs = 10000
```

Comment:
I like this to be smaller to reduce the chance that a logStartOffset updated by DeleteRecordsRequest is lost. This is useful e.g. if there is only one replica and the broker may crash. Anyway, I will change it to 60 sec.